tests/test_version.py

```diff
@@ -0,0 +1,38 @@
+import importlib
+import os
+import sys
+from unittest import TestCase
+
+
+class TestVersion(TestCase):
+    def test_version(self):
+        self.assertTrue(os.path.exists(os.path.join("montepy", "_version.py")))
+        # Test without version.py
+        try:
+            # try without setuptools_scm
+            old_file = os.path.join("montepy", "_version.py")
+            new_file = os.path.join("montepy", "_version.bak")
+            os.rename(old_file, new_file)
```

Comment:
This test was meant for covering people who clone the repo because `_version.py` is not version controlled and generated by `setuptools_scm`. 

I see your point about tests should work in a static way. Could we maybe make this so it isn't included in the packaged test suite, and only runs for developers?

Reply:
![Untitled](https://github.com/idaholab/MontePy/assets/19986691/0cae3d77-8547-4353-929c-4118550bb876)

Reply:
`MANIFEST.in` works like a charm:

```
copying tests/__init__.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/constants.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/test_cell_problem.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/test_data_inputs.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/test_edge_cases.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/test_geometry.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/test_importance.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/test_input_file.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/test_integration.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/test_interface.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/test_main.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/test_material.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/test_mcnp_problem.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/test_mode.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/test_numbered_collection.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/test_scripts.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/test_source_def.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/test_surfaces.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/test_syntax_parsing.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/test_tally.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/test_transform.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/test_universe.py -> montepy-0.2.7.dev15+g7caadf8/tests
copying tests/test_utilities.py -> montepy-0.2.7.dev15+g7caadf8/tests
```